modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/bayesian_uq_1d_diff.md

```diff
@@ -0,0 +1,207 @@
+    caption=Complete input file for master application for executing Bayesian UQ using the base MCMC class.
+
+### Available Sampling Algorithms
+
+Currently, multiple parallel MCMC samplers have been implemented in stochastic tools module, including the [PMCMCBase](/PMCMCBase.md), [IndependentGaussianMH](/IndependentGaussianMH.md), [AffineInvariantStretchSampler](/AffineInvariantStretchSampler.md), and [AffineInvariantDES](/AffineInvariantDES.md). The above example is demonstrated on [PMCMCBase](/PMCMCBase.md), which only performs a crude Monte Carlo simulation. Switching to an advanced MCMC sampler only requires minor changes in the [Samplers](Samplers/index.md) block and the [Reporters](Reporters/index.md) block in the master input file, while the input syntax for the sub-application remains the same. Three available MCMC samplers are explained below respectively:
+
+- [IndependentGaussianMH](IndependentGaussianMH.md): Performs Metropolis-Hastings MCMC sampling with independent Gaussian propoposals. Under the [Samplers](Samplers/index.md) block, `std_prop` specifies the standard deviations of the independent Gaussian proposals for making the next proposal. `seed_inputs` takes in a reporter named `mcmc_reporter` which reports seed inputs values for the next proposals. The `mcmc_reporter` is defined under the [Reporters](Reporters/index.md) block of type [IndependentMHDecision](reporters/IndependentMHDecision.md), which performs decision making for independent Metropolis-Hastings MCMC.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_imh.i block=Samplers
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_imh.i block=Reporters
+
+- [AffineInvariantStretchSampler](AffineInvariantStretchSampler.md): Performs affine-invariant ensemble MCMC with stretch sampler. Different from other samplers, it takes in two reporters to access the mean and variance of the previous state of all the walkers (`previous_state` and `previous_state_var`).  By default, the walkers take a step size of 0.2 in the stretch sampler. Correspondingly, an `mcmc_reporter` of type [AffineInvariantStretchDecision](reporters/AffineInvariantStretchDecision.md) need be defined in the [Reporters](Reporters/index.md) block for decision making.
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_ss.i block=Samplers
+
+!listing modules/stochastic_tools/test/tests/samplers/mcmc/main_ss.i block=Reporters
+
+- [AffineInvariantDES](AffineInvariantDES.md): Performs affine-invariant ensemble MCMC with differential sampler. Similar to the stretch sampler, the evolution sampler requires two reporters to access the mean and variance of the previous state of all the walkers (`previous_state` and `previous_state_var`). Correspondingly, an `mcmc_reporter` of type [AffineInvariantDifferentialDecision](reporters/AffineInvariantDifferentialDecision.md) need be defined in the [Reporters](Reporters/index.md) block for decision making.
```

Comment:
Cite the DES paper. Also ignore mentioning about `previous_state_var` if the sigma term is not inferred.

Reply:
okay